<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QWidget" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ChatWindow
{
background-color:white;
}
#messageGroupBox
{
border:none;
}
#messageEditBox
{
border:none;
background-color:#d5dfe6;
border-radius:3px;
color:#6d818f;
font-family: &quot;Proxima Nova Rg&quot;;
font-size:14px;
padding:7px;
}
#sendButton
{
background-color:#509ee2;
border:none;
border-radius:3px;
font-family: &quot;Proxima Nova Rg&quot;;
font-size:14px;
padding:14px;
color:white;
}
#sendButton:hover
{
background-color:rgb(75, 151, 213);
}
#scrollArea
{
background-color:transparent;
border:none;
}
#scrollAreaContents
{
background-color:transparent;
}
#scrollArea .QLabel
{
background-color:#509ee2;
font-family: &quot;Proxima Nova Rg&quot;;
color:white;
font-size:14px;
padding:10px;
margin-left:50px;
border-radius:3px;
}
#scrollArea .QLabel[my=&quot;true&quot;]
{
background-color:#f1f5f8;
margin-left:0px;
margin-right:50px;
color:#444444;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>408</width>
        <height>541</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="messagesVertialLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="messageGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPlainTextEdit" name="messageEditBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="plainText">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>SEND</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
